src/popup/components/TransactionLabelsList.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <div
+    class="transaction-tag-list"
+    :class="{ dense }"
+  >
+    <TransactionLabel
+      data-cy="label"
+      :tx="tx"
+      :is-incomplete="isIncomplete"
+      :is-pending="isPending"
+      :is-claim="isClaim"
+      :transaction-owner="transactionOwner"
+      :owner-name="ownerName"
+      :transaction-date="transactionDate"
+      :label-element="label"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  computed,
+  defineComponent,
+  PropType,
+} from '@vue/composition-api';
+import { SCHEMA } from '@aeternity/aepp-sdk';
+import { TranslateResult } from 'vue-i18n';
+import { useTransactionTx } from '../../composables';
+import { useGetter, useState } from '../../composables/vuex';
+import {
+  IAccount,
+  INetwork,
+  ITokenList,
+  ITx, Label, TransactionType,
+  TxFunctionRaw,
+} from '../../types';
+import {
+  FUNCTION_TYPE_DEX,
+} from '../utils';
+import TransactionLabel from './TransactionLabel.vue';
+
+export default defineComponent({
+  components: { TransactionLabel },
+  props: {
+    tx: { type: Object as PropType<ITx>, required: true },
+    transactionDate: { type: String, default: '' },
+    transactionOwner: { type: String, default: null },
+    isIncomplete: Boolean,
+    isPending: Boolean,
+    isClaim: Boolean,
+    dense: Boolean,
+  },
+  setup(props, { root }) {
+    const {
+      txType,
+      isAllowance,
+      isDex,
+    } = useTransactionTx({ store: root.$store, tx: props.tx });
+
+    const account = useGetter<IAccount>('account');
+    const activeNetwork = useGetter<INetwork>('activeNetwork');
+    const availableTokens = useState<ITokenList>('fungibleTokens', 'availableTokens');
+    const getTxDirection = useGetter('getTxDirection');
+    const accounts = useGetter('accounts');
+
+    const addComa = (text: TranslateResult) => `${text},`;
+    const labelWrapper = (text: TranslateResult): Label => ({ text });
+
+    const label = computed((): Label => {
+      const transactionTypes = root.$t('transaction.type') as Record<TransactionType, TranslateResult>;
+      const transactionListTypes = root.$t('transaction.listType') as Record<TransactionType, TranslateResult>;
```

Comment:
the types are wrong here - TransactionType is in PascalCase and the values returned from the translation file are in camelCase